upload.sh

```diff
@@ -525,13 +525,13 @@ _setup_arguments() {
                         ;;
                     root-dir)
                         _check_longoptions
-                        _check_default "${!OPTIND}" "ROOTDIR=${!OPTIND/default=/}" "UPDATE_DEFAULT_ROOTDIR=_update_config"
+                        ROOTDIR="${!OPTIND/default=/}"
+                        [[ ${!OPTIND} = default* ]] && UPDATE_DEFAULT_ROOTDIR="_update_config" || :
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Note that A && B || C is not if-then-else. C may run when A is true.](https://app.codacy.com/manual/labbots/google-drive-upload/pullRequest?prid=5565059)